docs/training/etl.md

```diff
@@ -3,7 +3,7 @@ title: Data Pipelines & ETL
 nav-id: etl
 nav-pos: 3
 nav-title: Data Pipelines & ETL
-nav-parent_id: tutorials
+nav-parent_id: training
 ---
 <!--
 Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This page (not in the diff) still has a few lines using "tutorial" instead of "training"